src/transformers_ocr.sh

```diff
@@ -51,19 +55,26 @@ if is_Xorg; then
 	if_installed maim xclip || exit 1
 
 	take_screenshot() {
-		maim --select --hidecursor --format=png --quality 1
+		maim --select --hidecursor --format=png --quality 1 > "$1"
 	}
 
 	print_platform() {
 		echo "Xorg"
 	}
 else
-	if_installed grim slurp wl-copy || exit 1
+	if is_GNOME; then
```

Comment:
Since GNOME is a separate *platform*, I think you can move this `if` outside.
e.g.,
```
if is_Xorg; then
    ...
elif is_GNOME; then
    ...
else
    ...
fi
```